application/aam-backend-service/src/main/resources/sql/schema.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE IF NOT EXISTS SYNC_ENTRY
```

Comment:
where is this used? can we use this into the module where it is internally relevant?

Reply:
used for internal h2 database. Will rename it for better understanding.